swagger_spec_validator/validator20.py

```diff
@@ -35,15 +35,16 @@ def validate_spec(spec_json, spec_url=None):
     :returns: `None` in case of success, otherwise raises an exception.
     :raises: :py:class:`swagger_spec_validator.SwaggerValidationError`
     """
-    validate_json(spec_json, 'schemas/v2.0/schema.json')
-
     # Dereference all $refs so we don't have to deal with them
     fix_malformed_model_refs(spec_json)
     spec_json = jsonref.JsonRef.replace_refs(spec_json,
                                              base_uri=spec_url or '')
     replace_jsonref_proxies(spec_json)
 
-    # TODO: Extract 'parameters', 'responses' from the spec as well.
```

Comment:
Doesn't look like it.  It is pretty obvious once you add a feature that requires those things.